osbenchmark/synthetic_data_generator/strategies/mapping_strategy.py

```diff
@@ -59,6 +109,23 @@ def __init__(self, mapping_config=None, seed=1):
             "geo_point": self.generate_geo_point,
         }
 
+    @staticmethod
+    def generate_fake_document(transformed_mapping: Dict[str, Callable]) -> Dict[str, Any]:
+        """
+        Generate a document using the generator functions
+
+        Args:
+            transformed_mapping: Dictionary of generator functions
+
+        Returns:
+            document containing lambdas that cna be invoked to generate data
```

Comment:
Are these generators called in a particular sequence?  What do they all do -- might want to document.  Okay, I see below -- they are for the various fields.
Also, cna -> can.